### DISCUSSION TITLE
Barotrauma should include a super simple mod by default to give new modders an example.

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#14522] [Ideas and suggestions]
Lots of people want to mod barotrauma but are afraid to start, and don't know exactly whats needed to start. ie the filelist.xml and general structure of a mod. 

I propose adding a super simple mod to barotraumas default install. Something like "Example mod" that just contains say, a new weapon, armor and/or holdable item or something. Basically just 1~3 items, simple as can be to show off filelist.xml. Maybe a text file to show off how translations work. Weapons/items don't really need to be unique, just an example. Like say a revolver that uses a sprite you just edited in photoshop to be darker, and does 10% more damage. Would show off how to load sprites from %moddir% too.

**Comment by SomeRandomNoobKekeke** [comment]
There's literally zero mods on workshop to look up to :BaroDev:
And you'll probably try to use some mods before making yours, so you could look up to their structure

What i didn't realize before i started modding is that you shouldn't just change game files, although it's much simpler and might even work game won't let you connect to other servers with that

You can just make one, and i'm sure :BaroDevs: will add it to modding guide, eventually
![Burgerpants3](https://github.com/user-attachments/assets/fd261c3f-66ea-4dce-9f74-73fe0de1541b)

**Reply by BlackMoons** [reply]
Many mods are kinda complicated and/or intimidating. Some people also don't want to 'rip off' others code. Even if it was just a 'basicbeginnermodExtractMe.zip' in the localmods directory, I think it would be worthwhile to include something.
